src/app/events/tasks.ts

```
@@ -278,6 +278,14 @@ ${remult.context.origin + '/s/' + task.editLink}
   },
   //@ts-ignore
   apiPrefilter: () => {
+    // Intentionally not checking site config. If drive tasks aren't enabled, this will show nothing.
+    //  TODO but preferrably we error like we would if allowApiRead were auth-only like before.
```

You can throw an exception - it'll be 500 and not 401, but it's good enough